omnibus/files/private-chef-cookbooks/private-chef/libraries/preflight_bookshelf_validator.rb

```diff
@@ -67,7 +67,7 @@ def verify_storage_type_unchanged
 
 Please set
 
-bookshelf['storage_type'] = :#{previous_value}
+bookshelf['storage_type'] = '#{previous_value}'
```

Comment:
Note that this is all inside a a HEREDOC.  So this is basically changing:

    ":#{previous_value}"

to 

    "'#{previous_value}'"